src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/pdl/model/Personalia.kt

```diff
@@ -72,8 +72,10 @@ data class Personalia(
     return this
   }
 
-  fun withForelderBarnRelasjon(barnFnr: String): Personalia {
-    forelderBarnRelasjon = listOf(ForelderBarnRelasjon(barnFnr, "barn", "forelder"))
+  fun withForelderBarnRelasjon(fnrBarn: List<String>): Personalia {
```

Comment:
Hvis dere gjør noe ala 
  fun withForelderBarnRelasjon(fnrBarn: List<String>): Personalia {
    val nyForeldreBarnRelasjon = fnrBarn.map { ForelderBarnRelasjon(it, "barn", "forelder") }
    forelderBarnRelasjon = nyForeldreBarnRelasjon
    return this
  }
Så kan dere unngå å bruke MutableList her.  Nå er ikke dette kritisk produksjonskode. Men greit å likevel å forsøke å unngå mutabilitet der man kan.